libs/sdk-ui-kit/src/Datepicker/Datepicker.tsx

```diff
@@ -43,19 +45,21 @@ interface IDatePickerState {
     focused: boolean;
 }
 
-function formatDate(date: Date, format: string, locale: string): string {
-    return moment(date).locale(locale).format(format);
+function formatDate(date: Date, dateFormat: string): string {
+    return format(date, dateFormat);
 }
 
-function parseDate(str: string, format: string, locale: string): Date | void {
-    const result = moment(str, format, locale, true);
-
-    if (result.isValid()) {
-        return result.toDate();
+function parseDate(str: string, dateFormat: string): Date | void {
```

Comment:
There is a very similar (same?) function in `DateRangePickerInputField.tsx`. Can we unify this?

Reply:
The same goes for the `formatDate` above.

Reply:
Skipping for now, no better place to put this and not worth creating a new package over.